src/pages/groups/GroupsPage.tsx

```diff
@@ -0,0 +1,295 @@
+                                updateGroupCallback(currentRowData.current.id, currentRowData.current.name, values);
+                            }
+                        },
+                    };
+                },
+            },
+        ],
+        [intl, usersOptions, updateGroupCallback]
+    );
+
+    const [rowsSelection, setRowsSelection] = useState<GroupInfos[]>([]);
+    const deleteGroups = useCallback((): Promise<void> | undefined => {
+        let groupNames = rowsSelection.map((group) => group.name);
+        return UserAdminSrv.deleteGroups(groupNames)
+            .catch((error) =>
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'groups.table.error.delete',
+                })
+            )
+            .then(() => gridContext?.refresh?.());
+    }, [gridContext, rowsSelection, snackError]);
+    const deleteGroupsDisabled = useMemo(() => rowsSelection.length <= 0, [rowsSelection.length]);
+
+    const addGroup = useCallback(
+        (group: string) => {
+            UserAdminSrv.addGroup(group)
+                .catch((error) =>
+                    snackError({
+                        messageTxt: `Error while adding group "${group}"${error.message && ':\n' + error.message}`,
+                        headerId: 'groups.table.error.add',
+                    })
+                )
+                .then(() => gridContext?.refresh?.());
+        },
+        [gridContext, snackError]
+    );
+    const { handleSubmit, control, reset, clearErrors } = useForm<{
+        group: string;
+    }>({
+        defaultValues: { group: '' }, //need default not undefined value for html input, else react error at runtime
+    });
+    const [open, setOpen] = useState(false);
+    const [showDeletionDialog, setShowDeletionDialog] = useState(false);
+    const handleClose = () => {
+        setOpen(false);
+        reset();
+        clearErrors();
+    };
+    const onSubmit: SubmitHandler<{ group: string }> = (data) => {
+        addGroup(data.group.trim());
+        handleClose();
+    };
+    const onSubmitForm = handleSubmit(onSubmit);
+
+    const handleRowClicked = useCallback((event: RowClickedEvent<GroupInfos>) => {
+        if (event.data?.name !== undefined) {
+            currentRowData.current = {
+                id: event.data?.id,
+                name: event.data?.name,
+                users: event.data?.users,
+            };
+        }
+    }, []);
+
+    return (
+        <Grid item container direction="column" spacing={2} component="section">
+            <Grid item container xs sx={{ width: 1 }}>
+                <GridTable<GroupInfos, {}>
+                    ref={gridRef}
+                    dataLoader={UserAdminSrv.fetchGroups}
+                    columnDefs={columns}
+                    defaultColDef={defaultColDef}
+                    //onCellEditingStopped={handleCellEditingStopped}
```

Comment:
```suggestion
                    stopEditingWhenCellsLoseFocus={true}
```
add this prop to exit editing when we click outside the select

Reply:
Done